Reworked a couple parts of the demo (#3)
groundlight/coffee_demo: audio/README.txt

> A placeholder folder for gtts generated audio files

Reviewer: ```suggestion
This directory will get audio files generated by `gtts`. 
```
